inst/@sym/chol.m

```diff
@@ -66,7 +66,15 @@
   elseif (nargin > 2)
     print_usage ();
   end
-  y = python_cmd('return _ins[0].cholesky(),', x);
+
+  cmd = { 'A = _ins[0]' ...
+          'try:' ...
+          '    y = A.cholesky_decomposition()' ...
+          'except AttributeError:' ...
+          '    y = A.cholesky()' ...
+          'return y' };
```

Comment:
Preference for without.

(Comma used to be required, someone convinced me to drop it, maybe @latot).